src/_common/Tab.js

```diff
@@ -21,13 +21,13 @@ export default class Tab extends Component {
     };
 
     render() {
-        const { imgSrc, selected, showIcon, showText, text, onMouseDown, onClose, closable } = this.props;
+        const { imgSrc, selected, showIcon, showText, text, onClick, onClose, closable } = this.props;
 
         return (
             <div
                 role="tab"
                 aria-selected={selected}
-                onMouseDown={onMouseDown}
+                onClick={onClick}
```

Comment:
Click = mouse down + mouse up if there was very little mouse move between the two.
I'd say Click is generally better and more widely used, but mouse down fires earlier and results in faster experience for the user. Click is fired after the mouse up.

It is generally OK to replace Mouse Down with Click anywhere where it is causing an issue. It is also generally good to try to do the reverse for other cases to see if it will feel faster and make a difference.